staging/src/kubevirt.io/client-go/kubevirt/typed/core/v1/virtualmachine_expansion.go

```diff
@@ -219,3 +220,20 @@ func (c *virtualMachines) ObjectGraph(ctx context.Context, name string, objectGr
 
 	return objectGraph, err
 }
+
+func (c *virtualMachines) EvacuateCancel(ctx context.Context, name string, evacuateCancelOptions *v1.EvacuateCancelOptions) error {
+	body, err := json.Marshal(evacuateCancelOptions)
+	if err != nil {
+		return err
+	}
+
+	return c.GetClient().Put().
+		AbsPath(fmt.Sprintf(vmSubresourceURLFmt, v1.ApiStorageVersion)).
+		Namespace(c.GetNamespace()).
+		Resource("virtualmachines").
+		Name(name).
+		SubResource("evacuate", "cancel").
+		Body(body).
+		Do(ctx).
+		Error()
+}
```

Comment:
The server side is configured to consume json `Consumes(mime.MIME_JSON).`.
We should explicitly set the header.
```golang
func (c *virtualMachines) EvacuateCancel(ctx context.Context, name string, evacuateCancelOptions *v1.EvacuateCancelOptions) error {
	body, err := json.Marshal(evacuateCancelOptions)
	if err != nil {
		return err
	}

	return c.GetClient().Put().
		AbsPath(fmt.Sprintf(vmSubresourceURLFmt, v1.ApiStorageVersion)).
		Namespace(c.GetNamespace()).
		Resource("virtualmachines").
		Name(name).
		SubResource("evacuate", "cancel").
		Body(body).
		SetHeader("Content-Type", "application/json").
		Do(ctx).
		Error()
}
```

Reply:
Reverted